dmapiclient/base.py

```diff
@@ -76,6 +76,9 @@ def _delete_with_updated_by(self, url, data, user):
     def _request(self, method, url, data=None, params=None):
         if not self.enabled:
             return None
+        if self.base_url is None:
+            logger.info("{} has no URL configured".format(self.__class__.__name__))
```

Comment:
This should never work like this in a production app on PaaS, so I don't think it makes sense to merge this into master since it hides a potential problem from monitoring and health checks. Adding search API to the PaaS environment is what we'd do when setting up an actual environment.

In the meantime, since we're running a throwaway prototype branch on PaaS, so I'm happy with any hack on that branch to fix issues caused by this, but I suggest we close this PR.